### PR TITLE
Fb login working

### PR DESCRIPTION
**Pull request by esiddi4**

### CODE REVIEW

**Comment by uuunk** [comment]
@esiddi4 build is failing. got a linter on here now so you'll need to run `rails check` before checking in. It'll autofix a few things but then you may need to manually fix some of the other errors that show up.

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -13,16 +13,19 @@
...
-  it { should validate_presence_of(:last_name) }
-  it { should validate_uniqueness_of(:fb_id) }
-  it { should validate_uniqueness_of(:email) }
+  it "should validate that email is case-sensitively unique" do
+    user = users(:bob_user)
+    new_user = User.new(first_name: "John", last_name: "Doe", email: user.email)
+    expect(new_user.valid?).to be_falsey
+    new_user.update_attribute(:email, user.email.upcase)
+    expect(new_user.valid?).to be_falsey
+  end
```
Comment (sanghun89) [inline spec/models/user_spec.rb]: YESSSSS

Comment (sanghun89) [reply spec/models/user_spec.rb]: that being said, if you would like to, can you give it a go for adding tests for the other user methods you have added?

File: app/assets/stylesheets/pages/login.scss [hunk 2]
```diff
@@ -50,6 +50,9 @@
     font-size: 1rem;
     color: white;
     background-color: $facebook-blue;
+    a {
+      color: white;
```
[the remark this answers is not in the document]
Comment (gcwokeup) [reply app/assets/stylesheets/pages/login.scss]: donezo

File: spec/models/user_spec.rb (hunk 1 above)
Comment (gcwokeup) [reply spec/models/user_spec.rb]: fosheez. on it

File: app/assets/stylesheets/layouts/footer.scss [hunk 3]
```diff
@@ -35,7 +35,7 @@
     }
 
     &__login {
-      color: white;
+      ;
```
Comment (sanghun89) [inline app/assets/stylesheets/layouts/footer.scss]: ruh roh removed too much here

File: public/422.html [hunk 4]
```diff
@@ -26,7 +26,7 @@
     border-top: #B00100 solid 4px;
     border-top-left-radius: 9px;
     border-top-right-radius: 9px;
-    background-color: white;
+    background-;
```
Comment (sanghun89) [inline public/422.html]: same here

Comment (gcwokeup) [reply public/422.html]: got it

File: app/assets/stylesheets/layouts/footer.scss (hunk 3 above)
Comment (gcwokeup) [reply app/assets/stylesheets/layouts/footer.scss]: got it

File: spec/models/user_spec.rb (hunk 1 above)
Comment (gcwokeup) [reply spec/models/user_spec.rb]: actually ill put this on another branch. We can use https://github.com/omniauth/omniauth/wiki/Integration-Testing

**Pull request merged**